src/file/file.ts

```diff
@@ -126,15 +129,16 @@ export class File {
    * @param fullPath full path of the file
    */
   async delete(podName: string, fullPath: string): Promise<void> {
-    assertActiveAccount(this.accountData)
+    assertAccount(this.accountData)
     assertFullPathWithName(fullPath)
     assertPodName(podName)
     const pathInfo = extractPathInfo(fullPath)
     const connection = this.accountData.connection
     const extendedInfo = await getExtendedPodsList(
       connection.bee,
       podName,
-      this.accountData.wallet!,
+      prepareEthAddress(this.accountData.wallet!.address),
```

Comment:
This statements repeats on a lot of places. It could be extracted to a private method?